generators/java/templates/libraries/jersey2/ApiClient.mustache

```diff
@@ -0,0 +1,1408 @@
+                                  new LinkedHashSet<>({{#enumValues}}{{#-first}}Arrays.asList({{/-first}}
+                                          "{{{.}}}"{{^-last}},{{/-last}}{{#-last}}
+                                  ){{/-last}}{{/enumValues}})
+                          )){{^-last}},{{/-last}}
+                  {{#-last}}
+                  ).collect(Collectors.toMap(Entry::getKey, Entry::getValue, (a, b) -> a, LinkedHashMap::new))
+                  {{/-last}}
+                  {{/variables}}
+          ){{^-last}},{{/-last}}
+  {{#-last}}
+  ){{/-last}}{{/servers}});
+  protected Integer serverIndex = 0;
+  protected Map<String, String> serverVariables = null;
+  {{^hasOperationServers}}
+  protected Map<String, List<ServerConfiguration>> operationServers = new HashMap<>();
+  {{/hasOperationServers}}
+  {{#hasOperationServers}}
+  protected Map<String, List<ServerConfiguration>> operationServers;
+
+  {
+    Map<String, List<ServerConfiguration>> operationServers = new HashMap<>();
+    {{#apiInfo}}
+    {{#apis}}
+    {{#operations}}
+    {{#operation}}
+    {{#servers}}
+    {{#-first}}
+    operationServers.put("{{{classname}}}.{{{operationId}}}", new ArrayList<>(Arrays.asList(
+    {{/-first}}
+            new ServerConfiguration(
+                    "{{{url}}}",
+                    "{{{description}}}{{^description}}No description provided{{/description}}",
+                    {{^variables}}
+                    new LinkedHashMap<>()
+                    {{/variables}}
+                    {{#variables}}
+                    {{#-first}}
+                    Stream.<Entry<String, ServerVariable>>of(
+                    {{/-first}}
+                            new SimpleEntry<>("{{{name}}}", new ServerVariable(
+                                    "{{{description}}}{{^description}}No description provided{{/description}}",
+                                    "{{{defaultValue}}}",
+                                    new LinkedHashSet<>({{#enumValues}}{{#-first}}Arrays.asList({{/-first}}
+                                            "{{{.}}}"{{^-last}},{{/-last}}{{#-last}}
+                                    ){{/-last}}{{/enumValues}})
+                            )){{^-last}},{{/-last}}
+                    {{#-last}}
+                    ).collect(Collectors.toMap(Entry::getKey, Entry::getValue, (a, b) -> a, LinkedHashMap::new))
+                    {{/-last}}
+                    {{/variables}}
+            ){{^-last}},{{/-last}}
+    {{#-last}}
+    )));
+    {{/-last}}
+    {{/servers}}
+    {{/operation}}
+    {{/operations}}
+    {{/apis}}
+    {{/apiInfo}}
+    this.operationServers = operationServers;
+  }
+
+  {{/hasOperationServers}}
+  protected Map<String, Integer> operationServerIndex = new HashMap<>();
+  protected Map<String, Map<String, String>> operationServerVariables = new HashMap<>();
+  protected boolean debugging = false;
+  protected ClientConfig clientConfig;
+  protected int connectionTimeout = 0;
```

Comment:
The default is 0? Perhaps we should increase this

Reply:
As in line 652, `0` means no timeout. If I recall well, that's the same value in onfido-node.
Perhaps it would make sense setting a default value not to 0.
Or at least showing how to customise it. 🤔

Reply:
Ah 🙈 yeah I think we should have a proper default and show how it can be customised

Reply:
Checked current values:
- [onfido-java](https://github.com/onfido/onfido-java/blob/e92cb1017a74fff58877f8a2ad0be06a7f4725d1/onfido-java/src/main/java/com/onfido/Onfido.java#L97) : read timeout 60s - can't find connection timeout
- [onfido-node](https://github.com/onfido/onfido-node/blob/v2.9.0/src/Onfido.ts#L49) : generic timeout set to 30s
- [onfido-ruby](https://github.com/onfido/onfido-ruby/blob/e3b62c0ca0b8406d74f1f51cb7b51d7ced404466/lib/onfido/options.rb#L7) : open timeout 10s read timeout 30s
- [onfido-python](https://github.com/search?q=repo%3Aonfido%2Fonfido-python%20timeout&type=code) : timeout is not set to any value (infinite)
- [api-php-client](https://github.com/search?q=repo%3Aonfido%2Fapi-php-client%20timeout&type=code) : no timeout setting found there (infinite)

I propose to put 30s as default timeout and leave instructions about how to change that. @DavidMealha-Onfido wdyt?

Reply:
Sounds good 👍